src/server/snapshot.h

```diff
@@ -141,6 +150,7 @@ class SliceSnapshot {
   DbIndex current_db_;
 
   std::unique_ptr<RdbSerializer> serializer_;
+  std::deque<DelayedEntry> delayed_entries_;  // collected during atomic bucket traversal
```

Comment:
why deque? vector (stack) won't work here?

Reply:
Actually, it will